exla/c_src/exla/exla.cc

```diff
@@ -2045,6 +2045,35 @@ ERL_NIF_TERM run(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]) {
   return term;
 }
 
+ERL_NIF_TERM device_assignment_to_device_id(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]) {
+  if (argc != 3) {
+    return exla::nif::error(env, "Bad argument count.");
+  }
+
+  exla::ExlaExecutable** exec;
```

Comment:
```suggestion
  exla::ExlaExecutable* exec;
```

Can't this be just a pointer?
Then on line 2056 you could pass `&exec` and use `exec->` instead of `(*exec)->` everywhere else